.vscode/settings.json

```diff
@@ -8,7 +8,7 @@
     "python.testing.pytestEnabled": true,
     "python.formatting.provider": "black",
     "python.languageServer": "Pylance",
-    "python.pythonPath": "/scratch/tmc43/pipenv/PandABlocks-client-0xhNhEPE/bin/python",
+    "python.pythonPath": "/home/giles/.local/share/virtualenvs/PandABlocks-client-5lyRsfiY/bin/python",
```

Comment:
Wish we could add a single line to a git ignore...

Reply:
Agreed - but the only affect is that after a pull you need to re-tell the IDE where the venv is and it is easily found in the dropdown.

Reply:
Someone has come up with an approach here but I think it would probably bite us worse than the current irritation at some point. https://stackoverflow.com/questions/16244969/how-to-tell-git-to-ignore-individual-lines-i-e-gitignore-for-specific-lines-of

Reply:
Sounds dangerous, I think we just stick with what we have...